packages/jsonresume-theme-standard/src/ui/Resume.js

```diff
@@ -29,7 +29,7 @@ const Resume = ({ resume }) => {
         }}
       />
       <link rel="preconnect" href="https://fonts.googleapis.com" />
-      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
I guess this was a mistake. Not sure that these links even works when not in the head.